### PR TITLE
Add motivation and publications to README

### PR DESCRIPTION
**Pull request by Park-Patrick**

### CODE REVIEW

**Comment by dorahermes** [comment]
Love the updated README! Only note that BIDS is now being extended beyond neuroimaging scans to electrophysiology data, so I would be more general in the project summary (maybe just neuroimaging and neurophysiology).

File: README.md [hunk 1]
```diff
@@ -53,8 +53,12 @@ alt="BIDS Starter Kit Video" width="480" height="360" border="10" /></a>
 - [Contributing](#contributing)
 - [Acknowledgements](#acknowledgements)
 
+## Motivation 
+The primary goal of this project is to simplify the process of learning about the Brain Imaging Data Structure (BIDS). We hope that the resources in this wiki, such as links to tutorials, easy-to-read documentation, and code will make BIDS easier to adopt. In order to remain up to date with the continuously changing BIDS specifications, we have adopted a similarly open model to allow [contributions](https://github.com/INCF/BIDS-Starter-Kit/blob/master/CONTRIBUTING.md) from the community.
```
Comment (KirstieJane) [inline README.md]: This is great! 👾

File: README.md [hunk 2]
```diff
@@ -53,8 +53,12 @@ alt="BIDS Starter Kit Video" width="480" height="360" border="10" /></a>
...
+
+
 ## Project Summary
-Neuroimaging scans can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but the many different specifications of these acquired images cause portability issues between different labs and scientists. BIDS (Brain Imaging Data Structure) is a framework for organizing data that standardizes file organization and dataset description between different researchers.
+Neuroimaging scans can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but the many different specifications of these acquired images cause portability issues between different labs and scientists. BIDS is a framework for organizing data that standardizes file organization and dataset description between different researchers.
```
Comment (KirstieJane) [inline README.md]: Suggest changing "Neuroimaing scans" to "Neuroimaging and neurophysiology data" based on @dorahermes' feedback.

File: README.md [hunk 3]
```diff
@@ -91,9 +95,24 @@ To find out more about all the different ways to contribute to the BIDS Starter
...
 ## Acknowledgements
-![incf](https://www.eudat.eu/sites/default/files/styles/medium/public/logo/INCF_0.png?itok=uRT54XCM)
-![gsoc](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1e/GSoC.png/220px-GSoC.png)
 
+### Sponsor Organizations
+
+<img src="https://www.eudat.eu/sites/default/files/styles/medium/public/logo/INCF_0.png?itok=uRT54XCM" width="200" height="125" /> <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1e/GSoC.png/220px-GSoC.png" width="125" height="125" />
+
+### Publications
+
```
Comment (KirstieJane) [inline README.md]: @chrisfilo - are there any other publications to mention here? the BIDS Apps one maybe?

File: README.md [hunk 4]
```diff
@@ -91,9 +95,24 @@ To find out more about all the different ways to contribute to the BIDS Starter
 If you're here during summer 2018 :icecream::palm_tree:, you should reach out to our lovely [Google Summer of Code][gsoc] student [Patrick Park][patrick-github]. It would really help his project along if you said hello and passed along any feedback you have :purple_heart:. Please don't be shy, **the newer you are the more valuable your feedback is** :thumbsup:
 
 ## Acknowledgements
-![incf](https://www.eudat.eu/sites/default/files/styles/medium/public/logo/INCF_0.png?itok=uRT54XCM)
-![gsoc](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1e/GSoC.png/220px-GSoC.png)
 
+### Sponsor Organizations
+
```
Comment (KirstieJane) [inline README.md]: @chrisfilo - at the moment we're only acknowledging INCF and GSOC ....should there be (Stanford Center for Reproducible Neuroscience)[http://reproducibility.stanford.edu) or anything/one else?

Comment (chrisfilo) [reply README.md]: (Laura and John Arnold Foundation)[http://www.arnoldfoundation.org/laura-and-john-arnold-foundation-announces-3-8-million-grant-to-stanford-university-to-improve-the-quality-of-neuroscience-research/], (National Institute for Mental Health)[http://grantome.com/grant/NIH/R24-MH114705-01], GSoC, INCF.

Comment (chrisfilo) [reply README.md]: This should really go to https://github.com/INCF/BIDS

Comment (KirstieJane) [reply README.md]: 👍 Thanks @chrisfilo - @Park-Patrick, please add these to the README 😄 

I don't think you link to pages from the images at the moment, but I like the idea of doing the links that Chris has added. You could link to the GSOC and INCF home pages too.

File: README.md (hunk 3 above)
Comment (chrisfilo) [reply README.md]: That's it for now.

**Comment by chrisfilo** [comment]
This is great content (thank you for adding this), but I wonder if it would not be better to put it on https://github.com/INCF/BIDS (bids.neuroimaging.io) since this is the place people first see.

File: README.md (hunk 4 above)
Comment (KirstieJane) [reply README.md]: @chrisfilo - sorry - I think our comments crossed.

Do you mean this section of the README should also be in the main BIDS repo? If yes, I think that's a great idea!

Comment (chrisfilo) [reply README.md]: yes

**Comment by KirstieJane** [comment]
@chrisfilo - could we have the info in both places?

**Comment by chrisfilo** [comment]
It's perfectly fine for this to be in two places. Most of the readme seems relevant and a bit overlapping with bids.neuroimaging.io. One bit that seems to be specific to bids-starter-kit is the contribution section.

**Comment by KirstieJane** [comment]
Oh - I think I've misunderstood - I though you were talking about adding it to the README at the github repo, but are you actually suggesting that we update the website @chrisfilo? 

We can move this conversation to a new issue (I can make it once I know the scope) 😄

**Comment by chrisfilo** [comment]
Yes I was referring to updating the bids.neuroimaging.io website (which resides at https://github.com/INCF/BIDS).

File: README.md (hunk 2 above)
Comment (Park-Patrick) [reply README.md]: okay done!

File: README.md (hunk 4 above)
Comment (Park-Patrick) [reply README.md]: Added the two other organizations and changed the linked the images to their respective websites as well ✅

**Pull request merged**